cwl/src/main/scala/cwl/CwlJsonToDelayedCoercionFunction.scala

```diff
@@ -62,6 +62,20 @@ private [cwl] object CwlJsonToDelayedCoercionFunction extends Json.Folder[Delaye
     case WomSingleFileType | WomMaybePopulatedFileType if value.toMap.get("class").flatMap(_.asString).contains("File") =>
       Json.fromJsonObject(value).as[File] match {
         case Left(errors) => errors.message.invalidNel
+        /*
+          From the CWL spec:
+          If no location or path is specified, a file object must specify contents with the UTF-8 text content of the file.
+          This is a "file literal". File literals do not correspond to external resources, but are created on disk with
+          contents with when needed for a executing a tool. Where appropriate, expressions can return file literals to
+          define new files on a runtime. The maximum size of contents is 64 kilobytes.
+
+          As this is <= 64 KB of data and is being put in a temp dir, the author doesn't anticipate needing a formal cleanup.
```

Comment:
the optimism is admirable

Reply:
Heh. My 2¢: I'm ok with this for the moment as long as an issue is filed and referenced here in the comments, similar to [this one](https://github.com/broadinstitute/cromwell/issues/3186).